CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -596,20 +596,17 @@ public function buildQuickForm() {
     // the is_monetary concept probably should be too as it can be calculated from
     // the existence of 'amount' & seems fragile.
     if ($this->_contributeMode == 'notify' ||
-      $this->_amount < 0.0 || $this->_params['is_pay_later'] ||
-      ($this->_separateMembershipPayment && $this->_amount <= 0.0)
+      $this->_amount <= 0.0 || $this->_params['is_pay_later']
```

Comment:
Worst case we'd end up with the wrong text on a button so this is pretty low risk!  What would we need to do to get rid of `_contributeMode` here?

Also, looks like the is_monetary comment no longer applies and should be removed.  Can the _contributeMode comment get a fixme/todo tag?